docs/AddingExecutionProvider.md

```diff
@@ -35,3 +35,17 @@ e.g.
   OrtReleaseObject(factory);
   OrtCreateSession(env, model_path, session_option, &session);
 ```
+
+# Testing your execution provider
+
+To ease the testing of your execution provider, you can add a new case for it to the `onnx_test_runner` command,
+do this by adding it to `onnxruntime/test/onnx/main.cc` file, following the pattern for other existing providers.
```

Comment:
The onnxruntime_perf_test can be updated as well, to help measure the perf of new EP
onnxruntime/test/perftest/command_args_parser.cc